recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ package:
   version: {{ version }}
 
 source:
-  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_nvdisasm/{{ platform }}/cuda_nvdisasm-{{ platform }}-{{ version }}-archive.{{ extension }}
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_nvdisasm/{{ platform }}/cuda_nvdisasm-{{ platform }}-{{ version }}-archive.{{ extension }}  # [linux64 or ppc64le or aarch64 or win]
```

Comment:
Ok this is the core of the issue. The re-rendering process does not like that `platform` is `undefined` in some cases

With `conda-build` version `24.5`, this was not an issue

However in `conda-build` version `24.7`, this is now an issue

Still need to dig further to understand what changed to cause this issue

Reply:
Have described this in more detail in issue: https://github.com/conda-forge/conda-smithy/issues/2011